src/internal/m365/graph/errors.go

```diff
@@ -131,7 +131,9 @@ func IsErrQuotaExceeded(err error) bool {
 }
 
 func IsErrExchangeMailFolderNotFound(err error) bool {
-	return hasErrorCode(err, resourceNotFound, mailboxNotEnabledForRESTAPI)
+	// Not sure if we can actually see a resourceNotFound error here. I've only
+	// seen the latter two.
```

Comment:
This is kind of an odd comment to leave.  Does it need a ticket?  Do you want to put your name on it in case someone has a question?

Reply:
I don't really know if I'd be of more help to someone if they did have questions about it. I didn't want to remove the `resourceNotFound` that was there previously because I wasn't sure if that would break some case I haven't been able to hit yet with my own testing